src/py42/clients/alerts.py

```diff
@@ -136,3 +136,28 @@ def get_aggregate_data(self, alert_id):
             :class:`py42.response.Py42Response`
         """
         return self._alert_service.get_aggregate_data(alert_id)
+
+    def get_all_alert_details(self, query):
+        """
+        Helper method that combines :func:`.search_all_pages()` and :func:`.get_details()`.
+        Returns an iterator of alert detail objects.
+
+        Args:
+            query (:class:`py42.sdk.queries.alerts.alert_query.AlertQuery`): An alert query.
+
+        Returns:
+            generator: An object that iterates over alert detail items.
+        """
+        query.page_size = 100
```

Comment:
Do we still want to document that `page_size` gets overriden?
What if a user is thinking they are managing their network usage by setting it

Reply:
yeah, that's a good call. I'll add a note.